### ISSUE TITLE
checking whether the C compiler works... no

### ISSUE THREAD

**Issue by jimpark** [Homebrew/homebrew#44405]
I've got my mac back from the Apple Store after a repair. This was a clean install with Yosemite where I installed Xcode 7 from the App Store and installed CLT using 'xcode-select --install'

Here is the output of installing python. But other products like macvim have similar issues:

```
jim$ brew install python
==> Downloading https://www.python.org/ftp/python/2.7.10/Python-2.7.10.tgz
Already downloaded: /Library/Caches/Homebrew/python-2.7.10.tgz
==> Downloading https://bugs.python.org/file30805/issue10910-workaround.txt
Already downloaded: /Library/Caches/Homebrew/python--patch-c075353337f9ff3ccf8091693d278782fcdff62c113245d8de43c5c7acc57daf.txt
==> Downloading https://bugs.python.org/file40479/xcode-stubs-2.7.patch
Already downloaded: /Library/Caches/Homebrew/python--patch-86714b750c887065952cd556f4d23246edf3124384f579356c8e377bc6ff2f83.patch
==> Patching
patching file Include/pyport.h
Hunk #1 succeeded at 713 (offset 14 lines).
Hunk #2 succeeded at 736 (offset 14 lines).
patching file Lib/distutils/ccompiler.py
patching file Lib/distutils/unixccompiler.py
==> ./configure --prefix=/usr/local/Cellar/python/2.7.10_2 --enable-ipv6 --datar
Last 15 lines from /Users/jim/Library/Logs/Homebrew/python/01.configure:
LDFLAGS=-isysroot 
MACOSX_DEPLOYMENT_TARGET=10.10

checking build system type... x86_64-apple-darwin14.5.0
checking host system type... x86_64-apple-darwin14.5.0
checking for --enable-universalsdk... no
checking for --with-universal-archs... 32-bit
checking MACHDEP... darwin
checking EXTRAPLATDIR... $(PLATMACDIRS)
checking for --without-gcc... yes
checking for gcc... clang
checking whether the C compiler works... no
configure: error: in `/private/tmp/python20150928-42030-1orh70/Python-2.7.10':
configure: error: C compiler cannot create executables
See `config.log' for more details
```

As you can see CLT is installed.
```
jim$ xcode-select -p
/Applications/Xcode.app/Contents/Developer
jim$ xcode-select -v 
xcode-select version 2339.
```

Here is the output of 'brew doctor':
```
jim$ brew config
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/homebrew
HEAD: 98b7a22d81cefba49dbf3dd03b76871e4ebdbe4e
Last commit: 74 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: 8-core 64-bit ivybridge
OS X: 10.10.5-x86_64
Xcode: 7.0
CLT: N/A
Clang: 7.0 build 700
X11: N/A
System Ruby: 2.0.0-p481
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: N/A
```

Here is the output of 'clang -v':
```
jim$ clang -v
Apple LLVM version 7.0.0 (clang-700.0.72)
Target: x86_64-apple-darwin14.5.0
Thread model: posix
```

Brew doctor only gives this warning:
```
jim$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Xcode 7 lacks the 10.10 SDK which can cause some builds to fail.
We recommend installing the Command Line Tools with:
  xcode-select --install
to resolve this issue.
```

Any help would be greatly appreciated!

**Comment by bfontaine** [comment]
Please run `xcode-select --install` as suggested by `brew doctor` and try again.

**Comment by jimpark** [comment]
I figured it out. Due to company policy, I have two accounts. One that is not a part of the admin group and one that is. I was just using login to get to my "admin" account to run brew commands and I was getting this error. I had run "xcode-select --install" many times but as my "user" account but it never fixed this problem.

I had to actually login as my "admin" account and run "xcode-select --install" and now it works fine. I hope this post will help others who are using a secondary "admin" account and a limited primary account.
